core/src/main/java/com/sequenceiq/cloudbreak/service/ClusterCommonService.java

```diff
@@ -258,4 +258,9 @@ public FlowIdentifier updateSalt(NameOrCrn nameOrCrn, Long workspaceId) {
         Stack stack = stackService.getByNameOrCrnInWorkspace(nameOrCrn, workspaceId);
         return clusterOperationService.updateSalt(stack);
     }
+
+    public FlowIdentifier updatePillarConfiguration(NameOrCrn nameOrCrn, Long workspaceId) {
+        Stack stack = stackService.getByNameOrCrnInWorkspace(nameOrCrn, workspaceId);
```

Comment:
please add `MDCBuilder.buildMdcContext(stack);` after this line. I missed it in case of `updateSalt`, I'm going to add it

Reply:
Added